src/server/internalServer.cpp

```diff
@@ -658,6 +658,8 @@ std::vector<std::string>
 InternalServer::search_catalog(const RequestContext& request,
                                kiwix::OPDSDumper& opdsDumper)
 {
+    // added for discussion in #530 using the following method we can deliver zim file irrespective of a remote URl present for it.
+    // auto filter = kiwix::Filter().valid(true).local(true);
```

Comment:
@kelson42 I have added that method in the comments here.
comment the line below it and remove comments from this one to get your desired results.

Reply:
@MananJethwani `kiwix-serve --port=8082 *zim` still gives me an empty welcome page.

Reply:
@kelson42 are you sure you commented the correct line? because I tried it and it works for me.
you need to comment L663 and uncomment L662 ... it works for me that way.
here is the  proof you can clearly see the last link in this entry is for thumbnail but if I would have added a URL there would be another link there.
![Screenshot from 2021-05-22 23-57-36](https://user-images.githubusercontent.com/50267986/119237339-ec8bcb00-bb59-11eb-8b24-b14752cec070.png)